core/storage/suggestion/gae_models.py

```diff
@@ -1990,3 +2000,1275 @@ def export_data(
+            accepted_translation_word_count=accepted_translation_word_count,
+            rejected_translations_count=rejected_translations_count,
+            first_contribution_date=first_contribution_date,
+            last_contribution_date=last_contribution_date)
+        entity.update_timestamps()
+        entity.put()
+        return entity_id
+
+    @staticmethod
+    def construct_id(
+        language_code: str, contributor_id: str
+    ) -> str:
+        """Constructs a unique ID for a
+        TranslationReviewerTotalContributionStatsModel instance.
+
+        Args:
+            language_code: str. ISO 639-1 language code.
+            contributor_id: str. User ID.
+
+        Returns:
+            str. An ID of the form:
+
+            [language_code].[contributor_id]
+        """
+        return (
+            '%s.%s' % (language_code, contributor_id)
+        )
+
+    # Here we use MyPy ignore because the signature of this method
+    # doesn't match with BaseModel.get().
+    # https://mypy.readthedocs.io/en/stable/error_code_list.html#check-validity-of-overrides-override
+    @classmethod
+    def get( # type: ignore[override]
+        cls, language_code: str, contributor_id: str
+    ) -> Optional[TranslationReviewerTotalContributionStatsModel]:
+        """Gets the TranslationReviewerTotalContributionStatsModel
+        matching the supplied language_code, contributor_id.
+
+        Returns:
+            TranslationReviewerTotalContributionStatsModel|None. The matching
+            TranslationReviewerTotalContributionStatsModel, or None
+            if no such model instance exists.
+        """
+        entity_id = cls.construct_id(
+            language_code, contributor_id)
+        return cls.get_by_id(entity_id)
+
+    @classmethod
+    def get_all_by_user_id(
+        cls, user_id: str
+    ) -> Sequence[TranslationReviewerTotalContributionStatsModel]:
+        """Gets all TranslationReviewerTotalContributionStatsModel
+        matching the supplied user_id.
+
+        Returns:
+            list(TranslationReviewerTotalContributionStatsModel). The matching
+            TranslationReviewerTotalContributionStatsModel.
+        """
+        return cls.get_all().filter(
+            cls.contributor_id == user_id
+        ).fetch(feconf.DEFAULT_SUGGESTION_QUERY_LIMIT)
+
+    @classmethod
+    def fetch_page(
+        cls,
+        page_size: int,
+        offset: int,
+        sort_by: Optional[str],
+        last_activity: Optional[int],
+        language_code: str
+    ) -> Tuple[Sequence[TranslationReviewerTotalContributionStatsModel],
+                int,
+                bool]:
+        """Returns the models according to values specified.
+
+        Args:
+            page_size: int. Number of models to fetch.
+            offset: int. Number of results to skip from the beginning of all
+                results matching the query.
+            sort_by: str|None. A string indicating how to sort the result.
```

Comment:
Same concerns as above, here and for other parts of this method.

Reply:
PTAL again. Thanks